### PR TITLE
Get /api/users

### PR DESCRIPTION
**Pull request by hedieh623**
request journey:
​
```mermaid
graph TD
    A>Request] -->|App| B(app.js)
    B --> |Controller| C(User.controller.js)
    C --> |Model| D(User.model.js)
    D --> E(db/connections.js)
    E --> F[(Database)]
```

### CODE REVIEW

**Comment by hedieh623** [comment]
[5. GET /api/users](https://trello.com/c/SqUDMhYb/19-5-get-api-users)

File: Controllers/Users.controller.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const selectUsers = require("../Models/Users.model.js");
+
+const getUsers = (req, res) => {
+  selectUsers().then((users) => {
+        res.status(200).send(users);
```
Comment (yalicee) [inline Controllers/Users.controller.js]: Needs to be on a key of users

**Review by yalicee — APPROVED** [review]
Make sure to change your tests and your controller with my comments in mind. Otherwise approved :)

File: Controllers/Users.controller.js (hunk 1 above)
Comment (hedieh623) [reply Controllers/Users.controller.js]: Im not sure if i know what you mean by this , could you clarify please?

**Review by Rayhaan2k9 — APPROVED** [review]
This one is great! Well done! 😄

File: Controllers/Article.controller.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
-const getArticle = (req, res) => {
+const getArticle = (req, res, next) => {
   selectArticle(req.params.article_id)
     .then((article) => {
       if (article) {
-        res.status(200).send({article:article});
+        res
+        .status(200)
+        .send({article:article});
```
Comment (hedieh623) [inline Controllers/Article.controller.js:9]: merge conflict gone wrong 🙃

**Pull request merged**